docs/content/docs/dev/table/materialized-table/quick-start.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Quick Start
+weight: 3
+type: docs
+aliases:
+- /dev/table/materialized-table/quick-start.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Quick Start Guide
+
+This guide will help you quickly understand and get started with materialized tables. It includes setting up the environment, creating materialized tables in CONTINUOUS mode, and creating materialized tables in FULL mode.
+
+## Environment Setup
+
+### Directory Preparation
+
+**Replace the example paths below with real paths on your machine.**
+
+- Create directories for Catalog Store and Catalog dependencies:
+
+```
+# Directory for File Catalog Store to save catalog information
+mkdir -p /path/to/catalog/store
+
+# Directory for test-filesystem Catalog to save table metadata and table data
+mkdir -p /path/to/catalog/test-filesystem
+
+# Default database for test-filesystem Catalog
+mkdir -p /path/to/catalog/test-filesystem/mydb
+```
+
+- Create directories for Checkpoints and Savepoints to save Checkpoints and Savepoints respectively:
+
+```
+mkdir -p /path/to/checkpoint
+
+mkdir -p /path/to/savepoint
+```
+
+### Dependency Preparation
+
+The method here is similar to the steps recorded in [local installation]({{< ref "docs/try-flink/local_installation" >}}). Flink can run on any UNIX-like operating system, such as Linux, Mac OS X, and Cygwin (for Windows). You need to have __Java 11__ installed locally. You can check the installed Java version with the following command:
```

Comment:
Java 8 also should work?

Reply:
Yes, but we need to avoid mentions of Java8 Because of https://issues.apache.org/jira/browse/FLINK-25247

Reply:
My question is: Why we emphasize Java 11 here? It would be cause user misunderstand java 8 can't work?

Reply:
Here I referenced the descriptions from other chapters. From my understanding, it is to inform users that Java 8 is no longer supported, and to discourage its use even though it is currently supported.